app/controllers/listings.controller.js

```diff
@@ -33,6 +33,22 @@ exports.list = function (req, res, next) {
     });
 };
 
+exports.allListingsForUser = function (req, res) {
+    var query = Listing.find({
+        $and: [
```

Comment:
You don't need the `$and` here because you're not anding any requirements. You can simply have `Listing.find({ user: req.params.userId });